tests/test_get_env.py

```diff
@@ -0,0 +1,16 @@
+from tests.mock_requests import MockResponse
+
+def test_info_v2_check(mock_api_v2_resource, utils_resource, mocker):
+    import afs
+    mocker.patch('afs.utils.urljoin')
+    from afs.get_env import AfsEnv
+    AfsEnv()
+    afs.utils.urljoin.assert_called_once_with('http://afs.org.tw/', extra_paths={})
```

Comment:
No assertion?
```python
afs_env = AfsEnv()
assert afs_env.api_version == 'v2'
```

Reply:
`afs.utils.urljoin.assert_called_once_with('http://afs.org.tw/', extra_paths={})
`
This line asserts the `AfsEnv` module inside is called the correct parameters or not.
If the function is wrong or called, the assert will be triggered.